app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Admin::UsersController < AdminController
+  def index
+    @users = User.all
+
+    respond_to do |format|
+      format.html
+    end
+  end
+
+  def approve
+    @user = User.find(params[:id])
+    @user.approve
+
+    respond_to do |format|
+      format.html { redirect_to admin_users_path, notice: 'Successfully approved user.' }
```

Comment:
Line is too long. [89/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.